65.md

```diff
@@ -6,59 +6,138 @@ Relay List Metadata
 
 `draft` `optional`
 
-Defines a replaceable event using `kind:10002` to advertise preferred relays for discovering a user's content and receiving fresh content from others.
+Defines two replaceable events to advertise preferred relays for discovering a user's content and receiving fresh content from others.
 
-The event MUST include a list of `r` tags with relay URIs and a `read` or `write` marker. Relays marked as `read` / `write` are called READ / WRITE relays, respectively. If the marker is omitted, the relay is used for both purposes.
+Relay Usages
+------------
+
+There are three defined usages for a relay:
+
+- `INBOX` relays are for events that the relay list owner will be reading, specifically watching for events that tag them. These do not include DMs (except in the deprecated form of relay list).
```

Comment:
This if this words them better: 

- `INBOX` relays store PUBLIC events that mention the relay list owner. Any user can publish and download such events from this relay.

- `OUTBOX` relays store events that the relay list owner has authored. Any user can publish and download such events from this relay

- `PRIVATE INBOX` relays store PRIVATE events that mention the relay list owner, such as perhaps direct messages (DMs) and giftwraps. Any user can publish such events to this relay, but only the owner can download them.

Reply:
My original intent was that INBOXES were "IN", not readable by anybody else.  Reading from other people's inboxes was your innovation.  I never did that or suggested that.

It has value though.  You can use this to find reactions to someone's post.

But it is bad for (for example) the chorus relay because if INBOX messages are readable, people can use the chorus relay as free disk space, and to place illegal content and the like.  Of course this is an out-of-scope issue for the nostr protocol. Maybe personal relays are a bad idea.

I'd like to hear from others on this.

Reply:
> Reading from other people's inboxes was your innovation. I never did that or suggested that.

Interesting, that's how I've always read it.

Reply:
Well, for me the difference between private and public inbox is that other people can download from the public inbox. Otherwise, they are the same.

Reply:
Yeah you are right.  I've been doing it wrong.  I'm not seeking replies from the right places. I'll have to update my code.

Reply:
I've pushed changes very close to your suggestion.